cmake/projects/Boost/hunter.cmake

```diff
@@ -442,7 +442,7 @@ if(MSVC)
   )
 endif()
 
-if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.3)
+if(NOT HUNTER_Boost_VERSION VERSION_LESS 1.72.0 AND CMAKE_VERSION VERSION_GREATER_EQUAL 3.3)
```

Comment:
could you add some brackets, so I don't have to look up the boolean rules if the `NOT` is evaluated before or after the `AND`

```cmake
if( (NOT HUNTER_Boost_VERSION VERSION_LESS 1.72.0) AND (CMAKE_VERSION VERSION_GREATER_EQUAL 3.3) )
```

Reply:
Fixed.

Reply:
thanks!